custom_components/tuya_local/devices/bauhn_5way_powerstrip_usb_AP5W-0624.yaml

```diff
@@ -0,0 +1,214 @@
+name: BAUHN 5-way powerstrip with USB
```

Comment:
This name should be without branding.  Also, I only see 5 switches and no timers, so is it 4 way powerstrip with USB, or 5 way without a controlled USB?

Reply:
Just to clarify this is a 5 way power board with 5 power outlets that are individually controllable, the USB ports are not controllable. I have removed the brand, and also the USB part from the name as it's irrelevant for the purpose of this config.